### ISSUE TITLE
YouTube Lectures

### ISSUE THREAD

**Issue by mikeizbicki** [mikeizbicki/cmc-csci145-math166#4]
The class youtube playlist is online at: https://www.youtube.com/playlist?list=PLSNWQVdrBwoZ3GE9SvH7dVBakCjjgD_R1

In general, I'll upload the zoom recordings to this playlist a few hours after class ends.

**Comment by mikeizbicki** [comment]
Today's lecture has finished uploading.  In general in the future, I won't announce when the lectures are done uploading, they'll just be sitting in the playlist waiting for you.

**Comment by mikeizbicki** [comment]
Zoom crashed in the middle of lecture on Wednesday 2020-09-24, and the video file from both before and after the crash is corrupted.  That means there's no lecture for this date on youtube.  Sorry :(

In class, we covered the instructions for your next programming assignment [03b_SLT_programming](https://github.com/mikeizbicki/cmc-csci145-math166/tree/fall2020/03b_SLT_programming), and we finished going over the SLT II notes.  I've posted a copy of the notes I completed during the lecture online at https://github.com/mikeizbicki/cmc-csci145-math166/blob/fall2020/03_statistical_learning_theory_ii/notes-lecture.pdf

If you have any questions about anything that we covered during that class (or any other class!), I'm happy to schedule 1-1 meetings to answer your questions.

**Comment by hanaknight** [comment]
I'm not sure if the notes version was updated yet?

**Comment by cnorth21** [comment]
The notes that were uploaded are blank :/

**Comment by mikeizbicki** [comment]
@sabhartono @cnorth21 Whoops, looks like I uploaded the wrong notes file.  The correct file should be uploaded now.

**Comment by mikeizbicki** [comment]
I'll be on zoom at 1:30pm tomorrow (Friday) for going over questions that people have about the missing lecture video.

**Comment by mikeizbicki** [comment]
Zoom crashed in the middle of lecture on Wednesday 2020-09-30.  The video file is corrupted again, so there's no lecture video to upload. Sorry :(

In class, we finished going over the model zoo notes and the `examples.py` python file.  I've uploaded the finished notes file to https://github.com/mikeizbicki/cmc-csci145-math166/blob/fall2020/04_model_zoo/notes-lecture.pdf and the finished `examples.py` file to https://github.com/mikeizbicki/cmc-csci145-math166/blob/fall2020/04_model_zoo/examples.py

I'll schedule a Friday office hours for data mining this week to go over any questions people have from the notes or from the homework due this weekend.  If you'd like to come, just reply with some times that work for you and I'll pick a time that seems to work well for everyone.
